hwdb/60-keyboard.hwdb

```diff
@@ -1114,6 +1114,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*90X3A*:pvr*
  KEYBOARD_KEY_96=!kbdillumup                            # Fn+F8 keyboard backlight up
  KEYBOARD_KEY_d5=!wlan                                  # Fn+F12 Wi-Fi toggle
 
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*900X5*:pvr*
+ KEYBOARD_KEY_89=!brightnessdown                        # Fn+F2 Backlight down
```

Comment:
It seems unusual to disable the auto-repeat for "brightness down", but not "brightness up" at the same time. Are you sure that this key works as intended? That would be a really weird BIOS bug to only break one key, but not the other.

Reply:
I checked again, you are right, it's working without the "!". I don't know why I did that.

Reply:
So you mean you can drop this line completely? It would be equally weird if you had to do define this key, but not brightnessup.

Reply:
@jcarrano: ping?

Reply:
Sorry, but I don't have the Samsung machine anymore, so I'm unable to test it right now. I guess the best is to just drop it.